docs/docs/modeling.md

```diff
@@ -110,6 +110,30 @@ classDiagram
 
 ## Task - タスク
 
+### Task
+
+- A specific work or activity that tenants are responsible for performing.
+
+### taskId
+
+- TaskId is the UUID
+
+- Constraints:
+
+### title
+
+- The owner can set a title as they prefer. This title can be modified at any time.
```

Comment:
We no longer use the word, owner, to represent the person who holds `ShareHouse`.
Therefore, could you please fix it?